torchvision/transforms/transforms.py

```diff
@@ -152,6 +156,7 @@ def __call__(self, pic):
         Returns:
             Tensor: Converted image.
         """
+        _log_api_usage_once(self)
```

Comment:
Can you create a constructor and move this there? It's not supposed to be on the __call__ or forward methods.